addons/mail/static/src/owl/components/composer/composer.xml

```diff
@@ -50,47 +51,59 @@
                     t-on-o-keydown-enter.stop="_onTextInputKeydownEnter"
                     t-on-paste="_onPasteTextInput"
                     t-ref="textInput"
                 />
-                <div class="o_Composer_buttons" t-att-class="{ 'o-mobile': storeProps.isMobile }">
-                    <div class="o_Composer_toolButtons" t-att-class="{ 'o-composer-has-current-partner-avatar': props.hasCurrentPartnerAvatar }">
-                        <div class="o_Composer_toolButtonSeparator"/>
-                        <EmojisButton
-                            class="o_Composer_button o_Composer_buttonEmojis o_Composer_toolButton"
-                            t-att-class="{ 'o-mobile': storeProps.isMobile }"
-                            t-on-o-emoji-selection.stop="_onEmojiSelection"
-                            t-ref="emojisButton"
-                        />
-                        <button class="o_Composer_button o_Composer_buttonAttachment o_Composer_toolButton btn btn-secondary fa fa-paperclip" t-att-class="{ 'o-mobile': storeProps.isMobile }" t-on-click="_onClickAddAttachment" title="Add attachment" type="button" aria-label="Add attachment"/>
+                <div class="o_Composer_buttons" t-att-class="{ 'o-composer-is-compact': props.isCompact, 'o-mobile': storeProps.isMobile }">
+                    <div class="o_Composer_toolButtons"
+                         t-att-class="{
+                         'o-composer-has-current-partner-avatar': props.hasCurrentPartnerAvatar,
+                         'o-composer-is-compact': props.isCompact,
+                         }">
+                        <t t-if="props.isCompact">
+                            <div class="o_Composer_toolButtonSeparator"/>
+                        </t>
+                        <div class="o_Composer_inputButtons">
+                            <EmojisButton
+                                class="o_Composer_button o_Composer_buttonEmojis o_Composer_toolButton"
+                                t-att-class="{ 'o-mobile': storeProps.isMobile }"
+                                t-on-o-emoji-selection.stop="_onEmojiSelection"
+                                t-ref="emojisButton"
+                            />
+                            <button class="o_Composer_button o_Composer_buttonAttachment o_Composer_toolButton btn btn-secondary fa fa-paperclip" t-att-class="{ 'o-mobile': storeProps.isMobile }" t-on-click="_onClickAddAttachment" title="Add attachment" type="button" aria-label="Add attachment"/>
+                        </div>
                         <t t-if="props.isExpandable">
-                            <button class="btn btn-secondary fa fa-expand o_Composer_button o_Composer_buttonFullComposer o_Composer_toolButton" t-att-class="{ 'o-mobile': storeProps.isMobile }" t-on-click="_onClickFullComposer" title="Full composer" type="button" aria-label="Full composer"/>
+                            <div class="o_Composer_otherButtons">
```

Comment:
This classname kinda sucks. what do you think of `o_Composer_secondaryToolButtons` instead? (Could therefore rename `o_Composer_inputButtons ` to `o_Composer_primaryToolButtons `)